olive/engine/engine.py

```diff
@@ -229,12 +229,12 @@ def run(
                     output_dir/output_footprints.json: footprint of the output models
 
                     A. One pass flow:
-                        output_dir/output_model/metrics.json: evaluation results of the output model
-                        output_dir/output_model/model_config.json: output model configuration
-                        output_dir/output_model/...: output model files
+                        output_dir/metrics.json: evaluation results of the output model
+                        output_dir/model_config.json: output model configuration
+                        output_dir/...: output model files
 
                     B. Multiple pass flows:
-                        output_dir/output_model/{pass_flow}/...: Same as A but for each pass flow
+                        output_dir/{pass_flow}/...: Same as A but for each pass flow
```

Comment:
since there is only one save_model call, I don't think this structure is correct? 
Please update the search and no search output model structure in this docstring after running and validating all scenarios.

Reply:
This case is same for both search and no search. Engine can still get a `output_footprints` and only one model will be saved to output folder. These are intermediate folders created by the engine. I can double check this and paste the folder structure later here.

Reply:
![image](https://github.com/user-attachments/assets/9f285905-9266-4ade-a950-a80e3f17e9ce)
They have metrics file in each pass flow pass folder. Though I don't think those files are needed so we can keep the output folder clean. We can remove those intermediate folders later if needed.